docs/user-guide/cli.md

```diff
@@ -5,7 +5,7 @@
 stylelint is an [npm package](https://www.npmjs.com/package/stylelint). Install it using:
 
 ```shell
-npm install -g stylelint
+npm install stylelint --save-dev save-prefix='~'
```

Comment:
What is `save-prefix='~'`?

Reply:
Oops, I thought [this config option](https://docs.npmjs.com/misc/config#save-prefix) was available as a flag. But it doesn't seem to be. I'll change it back.